src/main/java/com/wavefront/sdk/common/clients/WavefrontClient.java

```diff
@@ -118,11 +118,21 @@
      * @param server A server URL of the form "https://clusterName.wavefront.com" or "http://internal.proxy.com:port"
```

Comment:
we should also update this line to say proxy://

Reply:
see above